plugins/catalog-backend/src/service/NextEntitiesCatalog.ts

```diff
@@ -204,6 +204,29 @@ export class NextEntitiesCatalog implements EntitiesCatalog {
   }
 
   async removeEntityByUid(uid: string): Promise<void> {
+    // Clear the hashed state of the immediate parents of the deleted entity.
+    // This makes sure that when they get reprocessed, their output is written
+    // down again. The reason for wanting to do this, is that if the user
+    // deletes entities that ARE still emitted by the parent, the parent
+    // processing will still generate the same output hash as always, which
+    // means it'll never try to write down the children again (it assumes that
+    // they already exist). This makes the database never "heal" from accidental
+    // deletes.
+    await this.database<DbRefreshStateRow>('refresh_state')
+      .update({
+        result_hash: 'child-was-deleted',
+      })
+      .whereIn('entity_ref', function parents(builder) {
```

Comment:
Nicer with an arrow function here?

Reply:
we've been using functions in all other places and i actually really like to have them named 🤔